dockerfile

```diff
@@ -4,23 +4,26 @@ FROM python:3.10-slim
 # Install system dependencies
 RUN apt-get update && apt-get install -y git procps
 
+# Install uv
+RUN pip install uv
```

Comment:
in all docker files can we not add this command to this line directly?:  RUN uv pip install --no-cache-dir -r requirements.txt

Reply:
@kinshuksinghbist